charms/istio-pilot/src/charm.py

```diff
@@ -251,6 +295,13 @@ def reconcile(self, event):
         # so that we can report them at the end.
         handled_errors = []
 
+        # Call upgrade_charm in case there are new configurations that affect the control plane
+        # This is useful when there is a missing configuration during the install process
+        try:
+            self.upgrade_charm(event)
```

Comment:
Assuming we do keep this in the reconcile function, can we also update the reconcile docstring to mention istio will also be reconciled?